test/unit/rules/require-input-label-test.js

```diff
@@ -81,5 +83,41 @@ generateRuleTests({
         source: '<Input/>',
       },
     },
+    {
+      template: '<input aria-label="first label" aria-labelledby="second label">',
+      result: {
+        message: ERROR_MESSAGE_MULTIPLE_LABEL,
+        line: 1,
+        column: 0,
+        source: '<input aria-label="first label" aria-labelledby="second label">',
+      },
+    },
+    {
+      template: '<input id="label-input" aria-label="second label">',
+      result: {
+        message: ERROR_MESSAGE_MULTIPLE_LABEL,
+        line: 1,
+        column: 0,
+        source: '<input id="label-input" aria-label="second label">',
+      },
+    },
+    {
+      template: '<input id="label-input" ...attributes>',
+      result: {
+        message: ERROR_MESSAGE_MULTIPLE_LABEL,
+        line: 1,
+        column: 0,
+        source: '<input id="label-input" ...attributes>',
```

Comment:
[question] how is this a multiple label issue?

Reply:
I'm not sure about this one but the current rule does not report an error if `...attributes` is present.
So to stay coherent with the current rule I've decided to report an error if `...attributes` + something else is present.

Reply:
I think that since we can't determine what's going on if `...attributes` is present, we have to allow it to pass.

Reply:
@MelSumner and I fixed this one up...